src/components/Pages/HomeSections/footer/footerStyle.scss

```diff
@@ -1,11 +1,12 @@
+$BackgroundColor: #81a0d3;
 .footer {
     width: 100%;
     height: fit-content;
     display: flex;
     flex-direction: column;
     align-items: center;
     position: relative;
-    background-color: rgb(117, 169, 228);
+    background-color: $BackgroundColor;
```

Comment:
General colors that will be later reused in other CSS files should be defined in globals.scss